### DISCUSSION TITLE
Remote Thermostat for CW400

### DISCUSSION THREAD

**Discussion by Halbamy** [emsesp/EMS-ESP32#2067] [Q&A]
Hi,

I'm struggeling to get a "remote thermostat" to work. I have a CW400 installed.
The correct way would be to use a RC 100 (RF). But I cannot choose this one. Wich one should I choose?

Thanks.

**Comment by proddy** [comment]
which version of EMS-ESP are you on?

**Reply by Halbamy** [reply]
3.6.5

**Reply by Halbamy** [reply]
I tried RC200 so far. Remote Temperatur is set vie MQTT but then currtemp for hc1 disappears.

`{
    "datetime": "30.09.2024 11:38",
    "intoffset": 0.0,
    "floordry": "off",
    "dampedoutdoortemp": 13.7,
    "floordrytemp": 0,
    "building": "medium",
    "minexttemp": -10,
    "damping": "on",
    "wwmode": "own prog",
    "wwcircmode": "off",
    "wwchargeduration": 15,
    "wwcharge": "off",
    "wwextra1": 0,
    "wwdisinfecting": "off",
    "wwdisinfectday": "su",
    "wwdisinfecttime": 60,
    "hc1": {
        "seltemp": 18.0,
        "haclimate": "selTemp",
        "mode": "manual",
        "modetype": "comfort",
        "ecotemp": 16.0,
        "manualtemp": 18.0,
        "comforttemp": 18.0,
        "summertemp": 17,
        "designtemp": 50,
        "offsettemp": 0,
        "minflowtemp": 35,
        "maxflowtemp": 55,
        "roominfluence": 3,
        "roominflfactor": 3.8,
        "curroominfl": 0.0,
        "nofrostmode": "room",
        "nofrosttemp": 5,
        "targetflowtemp": 23,
        "heatingtype": "radiator",
        "summersetmode": "winter",
        "summermode": "winter",
        "controlmode": "weather compensated",
        "program": "prog 1",
        "tempautotemp": -1.0,
        "remoteseltemp": 0.0,
        "fastheatup": 0,
        "switchonoptimization": "on",
        "reducemode": "room",
        "noreducetemp": -31,
        "reducetemp": 0,
        "wwprio": "off",
        "coolingon": "on",
        "hpmode": "heating",
        "control": "RC200",
        "remotetemp": 20.0
    }
}`

**Reply by proddy** [reply]
any chance you could try with firmware version 3.7.0 which is in dev? I know Michael made some more improvements to the remote thermostat code

**Reply by Halbamy** [reply]
Yes. Maybe it's worth a try. I noticed that the remote thermostat did not popup in the device list on the ems-esp webpage

**Reply by Halbamy** [reply]
I had some time to test the latest dev release and it seems that the remote thermostat is workung. at least I can see the remote temperature in the HA climate instance.
